src/server/handlers/launchpad.js

```diff
@@ -83,6 +83,14 @@ const RESPONSE_SNAPCRAFT_YAML_NO_NAME = {
   }
 };
 
+const RESPONSE_SNAP_NAME_NOT_REGISTERED = {
+  status: 'error',
+  payload: {
+    code: 'snap-name-not-registered',
+    message: 'Snap name is not registered in the store'
```

Comment:
Yes, it should be handled in views, as HTML in such messages would be escaped by React.

Sure, ideally we want actionable error message like this, but for MVP demo this should be enough. And it would be great to have some designs if we start to look into details like this.